libc/include/limits.h

```diff
@@ -0,0 +1,16 @@
+#ifndef __LIMITS_H
+#define __LIMITS_H
+
+/* Maximum number of bytes in a pathname, including the terminating null. */
+#define PATH_MAX 128
+
+/* Maximum number of bytes in a filename, not including terminating null. */
+#define NAME_MAX 14
+
+#define PIPE_BUF 80
+
+#define OPEN_MAX 20
```

Comment:
Lets indicate OPEN_MAX comes from linuxmt/fs.h in a comment, along with linuxmt/config.h for PIPE_BUF. I know see why you're talking about including config.h. I think PIPE_BUFSIZ got rammed into config.h as a tunable parameter, but thinking again, it doesn't belong there, and probably ought to be moved to linuxmt/fs.h.